op-acceptance-tests/tests/interop/interop_test_helpers.go

```diff
@@ -238,6 +239,10 @@ func ValidateEveryMessage(ctx context.Context, opt txplan.Option, dependOn *txin
 		indexes = append(indexes, idx)
 	}
 	tx.Content.Fn(txintent.ExecuteIndexeds(constants.MultiCall3, constants.CrossL2Inbox, &dependOn.Result, indexes))
+	// We need to wait for a block with a timestamp greater than the timestamp of `dependOn.PlannedTx.IncludedBlock`.
+	// TODO this is ugly... we may need an alternative to `txplan.WithAgainstLatestBlock`.
+	// Maybe `txplan.WithTimestampInvariant(client AgainstLatestBlock, initMsgBlock plan.Lazy[eth.BlockInfo])`?
+	time.Sleep(time.Second * 2)
```

Comment:
Question for reviewers: this `time.Sleep` feels out of place. Should I create the `txplan.WithTimestampInvariant` or leave it be for now? Maybe there's a better way than creating another `txplan.Option`.

Reply:
Could use [`WaitForBlock()`](https://github.com/ethereum-optimism/optimism/blob/develop/op-e2e/e2eutils/geth/wait.go#L116) with a calculated block number based on the chain's blocktime and current block number.

Reply:
Also spoke in person about this -- it should not be required to wait for a later timestamp to submit executing messages, they should be valid on the same timestamp as the initiating message.

Reply:
Referencing the extended async discussion in Discord [here](https://discord.com/channels/1244729134312198194/1370463830374879353)